modules/templates-rails-creating-frontend-service.adoc

```diff
@@ -48,7 +48,7 @@ env": [
         "value": "postgresql"
     }
 
-],
+]
```

Comment:
Only thing I am not sure about is editing this output. You'll need QE to verify. Fixing _Example output_ looks good to me.

Reply:
+1
If you're changing the contents of a codeblock, you need to get QE to confirm. If you had only changed line 30, we could merge.

Reply:
Sorry about that. I did that by mistake. I changed it back to the original.